### ISSUE TITLE
10.0 lua error

### ISSUE THREAD

**Issue by Derployd** [GurliGebis/AngrierWorldQuests#23]
Received the following lua error after using a hearthstone.
```
Message: [ADDON_ACTION_BLOCKED] AddOn 'AngrierWorldQuests' tried to call the protected function 'MultiBarBottomLeft:SetPointBase()'.
Time: Wed Nov  9 10:51:48 2022
Count: 1
Stack: [ADDON_ACTION_BLOCKED] AddOn 'AngrierWorldQuests' tried to call the protected function 'MultiBarBottomLeft:SetPointBase()'.
[string "@Interface/FrameXML/EditModeManager.lua"]:629: in function `UpdateBottomActionBarPositions'
[string "@Interface/FrameXML/EditModeManager.lua"]:555: in function `UpdateActionBarLayout'
[string "@Interface/FrameXML/EditModeSystemTemplates.lua"]:655: in function `UpdateGridLayout'
[string "@Interface/FrameXML/ActionButton.lua"]:373: in function `UpdateAction'
[string "@Interface/FrameXML/ActionButton.lua"]:803: in function `OnEvent'
[string "@Interface/FrameXML/ActionButton.lua"]:216: in function <Interface/FrameXML/ActionButton.lua:213>

Locals: <none>
```

This error occurred at the same time as the following errors with Blizzard's EditMode;

```
Message: Interface/FrameXML/EditModeSystemTemplates.lua:631: attempt to perform arithmetic on local 'offsetY' (a nil value)
Time: Wed Nov  9 10:51:48 2022
Count: 9
Stack: Interface/FrameXML/EditModeSystemTemplates.lua:631: attempt to perform arithmetic on local 'offsetY' (a nil value)

Locals:
``` 

```
Message: Interface/FrameXML/EditModeUtil.lua:22: attempt to perform arithmetic on local 'offsetX' (a nil value)
Time: Wed Nov  9 10:51:48 2022
Count: 6
Stack: Interface/FrameXML/EditModeUtil.lua:22: attempt to perform arithmetic on local 'offsetX' (a nil value)

Locals: 
```

I am aware that these may be unrelated, but I've included them just in case there is a cause/effect issue.

This is not 100% repeatable after hearthing, and I understand there are a lot of taint issues with the new interface and many mods, with a lot of false positive lua reports, which this could very well be.

**Comment by Derployd** [comment]
Ok, so I just received an identical error as per my first post, only now Immersion is being listed as the violating addon. So I think its safe to say this isn't caused by Angrier World Quests and is very much an issue with how I've placed one of my action bars.

**Comment by GurliGebis** [comment]
I'm pretty sure this is something inside WoW itself, that is broken.
I see it for several different addons, so my guess is that it is something else that is broken, and will be fixed with 10.0.2

**Comment by jleafey** [comment]
Thanks for giving it a look!  I fear you are correct, the cause of these exceptions is likely a bit deeper in the "stack", which basically leaves WoW's code as the suspect.  I try to use a pretty consistent batch of addons across my dozen or so alts, but there are variations.  Since the apparent sources of the exceptions seems to vary, I can only guess that the issue is buried somewhere in the "lowest common denominator".
